### DISCUSSION TITLE
Documentation site proposal

### DISCUSSION THREAD

**Discussion by geopor** [tearust/teaproject#128] [General]
We're in the process of moving our documentation site off of Github for a variety of reasons:

- Github's wiki only allows alphabetical ordering of the sidebar (we can't shape the navigational menus in any way)
- No search
- Very SEO-unfriendly .. none of the content here is indexed by Google

We're moving to https://tearust.github.io/teaproject/#/

This is our proposed design of the new documentation site:

## Getting Started
- **TEA Mining**
- **Stakers / Investors**
- **Consumers / Normal Users**

**Comment by geopor** [comment]
The top level categories and sub-categories are up for discussion. Please leave your input here as well as what you most want out of a documentation site.

And please vote in our Telegram group about which kind of doc site you find most useful: https://t.me/teaprojectorg/17335

**Comment by geopor** [comment]
Looks like everyone (5 votes) so far wants a practically oriented site:

<img width="403" alt="Screen Shot 2022-07-13 at 8 59 07 AM" src="https://user-images.githubusercontent.com/86096370/178778725-1c5e9b07-7f2e-4ed2-a48a-d484c3473f94.png">

**Reply by kevingzhang** [reply]
No doubt, the question itself has told the user which to vote.
